packages/ui/src/components/va-carousel/VaCarousel.vue

```diff
@@ -336,7 +336,7 @@ export default defineComponent({
     }
   }
 
-  .va-image {
+  .va-aspect-ratio {
```

Comment:
It must stay va-image, the same name as component have.

so it is 

<div class="va-image va-aspect-ration">